users/autostart.rst

```diff
@@ -404,6 +404,13 @@ How to set up a user service
    Otherwise the user service will not start, because by default, systemd checks for user
    services before your home directory has been decrypted.
 
+Automatic start-up of systemd user instances at boot (before login) is possible
+through systemd's "lingering" function, if a system service cannot be used
+instead.  Refer to the `loginctl enable-linger`__ command to allow this for a
+particular user.
+
+__ https://www.freedesktop.org/software/systemd/man/loginctl.html#enable-linger%20USER%E2%80%A6
```

Comment:
New link syntax - how does that work if there's more than one? Maybe lets keep using the `text <url>`__ one just to not add even more variability to the syntax?

Reply:
It's called an anyonymous indirect link IIRC and works only once, locally.  Having this long URL inline looked ugly and I think it disturbs the reading flow when editing the document.  We can switch to a regular indirect link (not anonymous) instead.

Reply:
I'd appreciate that.

Reply:
Done, sorry for the delay.